front_end/dart_tests/context_for_evaluation.dart

```diff
@@ -0,0 +1,24 @@
+// Context for tests to exercise evaluating expressions.
+
```

Comment:
add a TODO calling out additional dart specific edge cases that should be tested here.
For example:
imports referenced with a prefix, names for cases where DDC has to perform renaming.

Reply:
Done, in the main test file.